graf2d/gpadv7/inc/ROOT/RAttrAxis.hxx

```diff
@@ -46,10 +46,11 @@ class RAttrAxis : public RAttrBase {
    RAttrValue<RPadLength> fTicksSize{this, "ticks_size", 0.02_normal};   ///<! ticks size
    RAttrValue<RColor> fTicksColor{this, "ticks_color", RColor::kBlack};  ///<! ticks color
    RAttrValue<int> fTicksWidth{this, "ticks_width", 1};                  ///<! ticks width
-   RAttrText fLabelsAttr{this, "labels"};                                ///<! text attributes for labels
+   RAttrValue<bool> fNoLabels{this, "nolabels", false};                  ///<! disable labels drawing
```

Comment:
Why not `fDrawLabels`? `!fNoLabels` is harder for my brain :-)

Reply:
I trying to have attributes which default value is `false`.

Reply:
OK, then maybe `fHideLabels`? That's still less negations, even if it's the same meaning IIUC?